packages/common/src/scorekeeper/scorekeeper.ts

```diff
@@ -235,6 +219,14 @@ export default class ScoreKeeper {
     return true;
   }
 
+  startJobs(metadata: JobRunnerMetadata) {
+    this._jobs = jobConfigs.map((config) => {
```

Comment:
is there a way to control which job to start, instead of all of them ? good for local testing purposes 
With the JobFactory you could control that by commenting out the specif job you didn't want

Reply:
The best would be to control the jobs from the config. For example, with `isActive` flag.
As for the `JobFactory`, with the current change, you can comment lines related to a particular `JobConfig` and achieve the same result. But of course, it's not how it should be in the first place. Config or env variables is the way to go.

```
  // {
  //   jobKey: "validity",
  //   defaultFrequency: Constants.VALIDITY_CRON,
  //   jobFunction: validityJobWithTiming,
  //   name: JobNames.Validity,
  // },
  {
    jobKey: "score",
    defaultFrequency: Constants.SCORE_CRON,
    jobFunction: scoreJobWithTiming,
    name: JobNames.Score,
  },
  ```